02.calendar/cal.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+require 'date'
+require 'optparse'
+require 'debug'
+
+def main
+  year = Date.today.year
+  month = Date.today.month
+  if ARGV.any?
+    opt = OptionParser.new
+    opt.on("-y [YYYY]", "Specify a year") {|value| year = value.to_i}
+    opt.on("-m [MM]", "Specify a month") {|value| month = value.to_i}
+    opt.parse!
+  end
+  puts "      #{month}月 #{year}"
+  puts "日 月 火 水 木 金 土"
+  print_cal(year, month)
+end
+
+def print_cal(year, month)
+  first_day = Date.new(year, month, 1)
+  last_day = Date.new(year, month, -1)
+  print " ".rjust(first_day.wday*3) # 一行目のpadding
```

Comment:
```suggestion
  print " ".rjust(first_day.wday * 3) # 一行目のpadding
```

スペースを入れないとRuboCopに怒られませんでした？
（再提出前は毎回RuboCopを通しましょう）

あと、コメントは「どうしても」というとき以外は付けなくても大丈夫です。
https://qiita.com/jnchito/items/f0d90af4ed44b7484103

Reply:
> スペースを入れないとRuboCopに怒られませんでした？
（再提出前は毎回RuboCopを通しましょう）

すいません、カレンダープログラムはまだRuboCopを使わないんでしたね。
なので、`*`の前後にスペースを入れるだけでOKです！